core/src/main/java/org/ehcache/core/statistics/TierStatistics.java

```diff
@@ -23,28 +23,96 @@
  */
 public interface TierStatistics {
 
+  /**
+   * List of statistics tracked on this cache
+   *
+   * @return a map of statistics per name
+   */
   Map<String, TypedValueStatistic> getKnownStatistics();
 
   /**
    * Reset the values for this tier. However, note that {@code mapping, maxMappings, allocatedMemory, occupiedMemory}
-   * but be reset since it doesn't make sense.
+   * won't be reset since it doesn't make sense.
    * <p>
    * <b>Implementation note:</b> Calling clear doesn't really clear the data. It freezes the actual values and compensate
    * for them when returning a result.
    */
   void clear();
```

Comment:
just for our knowledge: who can call clear and thus reset the stats ? (fyi @anthonydahanne - clear impacts for example hit count, update count, etc)